src/components/Tags/Tags.stories.js

```diff
@@ -0,0 +1,39 @@
+import Tags from './Tags.vue'
+
+export default {
+    title: 'Miter Design/Tags',
+    component: Tags,
+    args: {
+        content: 'Tags'
+    },
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/HJgj2eHBaqt1SAYruZNjQQ/Prefect?node-id=1080%3A4411'
+        }
+    }
+}
+
+const Template = (args) => ({
+    components: { Tags },
+    setup() {
+        return { args }
+    },
+    template: '<Tags v-bind="args" />'
+})
+
+export const Default = Template.bind({})
+
+export const EmptyInternalTags = Template.bind({})
+EmptyInternalTags.args = {
+    tags: [],
+    icon: 'pi-label',
+    flat: true
+}
+
+export const InternalTags = Template.bind({})
```

Comment:
Lets change the names of these stories. `InternalTags` is an implementation detail and not something we should use for examples. Maybe just `Empty State` and `Default`

Reply:
What about the one that has tags passed? Call it `State With Values`?

Reply:
I think `Default` is an appropriate name and one we've used for other stories as the default implementation. I'd also put that one first in the list.